packages/common/addon/services/web3.js

```diff
@@ -0,0 +1,17 @@
+import Service from '@ember/service';
+import Web3 from 'web3';
+
+export default Service.extend({
+  isMetamask: false,
+
+  init() {
+    this._super(arguments);
+
```

Comment:
According to some documentation I found, you'll need to do this to enable metamask to work with your domain (as well as it will return the current address in the promise as soon as the user approves connection request). https://github.com/cardstack/tally/blob/0c054d04e3cf6fa5db9a06e2e98c97b63480ee3e/packages/common/addon/services/web3.js#L44

<img width="429" alt="Screen Shot 2019-06-17 at 3 11 59 PM" src="https://user-images.githubusercontent.com/61075/59630049-57ccf700-9112-11e9-8d9a-d873a47aefb1.png">

Reply:
This is great, didn't even know `ethereum.enable()` was a thing. I had punted on dealing with it for now, I was relying on the user having disabled Privacy Mode (which is clearly not optimal). Yoinking all of this :)